### PR TITLE
Support multiple redirect_uri values

### PR DESCRIPTION
**Pull request by zachmargolis**
**Why**: Better OIDC support

First step of multiple, eventually will follow up with a PR to drop the old column once we have everything running off the new column

### CODE REVIEW

File: db/migrate/20170531194154_add_multiple_redirect_uris.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class AddMultipleRedirectUris < ActiveRecord::Migration
+  def up
+    add_column :service_providers, :redirect_uris, :json
```
Comment (monfresh) [inline db/migrate/20170531194154_add_multiple_redirect_uris.rb]: Why JSON and not Array?

Comment (zachmargolis) [inline db/migrate/20170531194154_add_multiple_redirect_uris.rb]: I was matching the style of the codebase already, there's a pre-existing JSON column but no other array columns:

```
identity-dashboard:master> git grep array -- db/migrate/
identity-dashboard:master> git grep json -- db/migrate/
db/migrate/20170117222117_add_attribute_bundle.rb:    add_column :service_providers, :attribute_bundle, :json
```

Comment (monfresh) [inline db/migrate/20170531194154_add_multiple_redirect_uris.rb]: I wouldn't consider choice of column types a style decision. Wouldn't it make sense to choose the right column type for the job?

Comment (zachmargolis) [inline db/migrate/20170531194154_add_multiple_redirect_uris.rb]: I think it's acceptable as is, how strongly do you feel that it should be changed to array?

Comment (monfresh) [inline db/migrate/20170531194154_add_multiple_redirect_uris.rb]: Not strongly.

**Pull request merged**